src/server/protocol.ts

```diff
@@ -3339,14 +3339,15 @@ namespace ts.server.protocol {
         insertSpaceAfterConstructor?: boolean;
         insertSpaceAfterKeywordsInControlFlowStatements?: boolean;
         insertSpaceAfterFunctionKeywordForAnonymousFunctions?: boolean;
-        insertSpaceAfterOpeningAndBeforeClosingEmptyBraces?: boolean;

```

Comment:
We need to leave this in and mark it deprecated instead. (And it needs to continue to work, in absence of the new rules, so that new TS versions don’t break the formatting of old VS versions.)

Reply:
@andrewbranch Thanks for the feedback. I've reverted `insertSpaceAfterOpeningAndBeforeClosingEmptyBraces`.